typing_inspect.py

```diff
@@ -279,14 +334,32 @@ def get_last_args(tp):
     if NEW_TYPING:
         raise ValueError('This function is only supported in Python 3.6,'
                          ' use get_args instead')
-    if is_classvar(tp):
+    elif is_classvar(tp):
         return (tp.__type__,) if tp.__type__ is not None else ()
-    if (
-        is_generic_type(tp) or is_union_type(tp) or
-        is_callable_type(tp) or is_tuple_type(tp)
-    ):
+    elif is_generic_type(tp):
+        try:
+            if tp.__args__ is not None and len(tp.__args__) > 0:
+                return tp.__args__
+        except AttributeError:
+            # python 3.5.1
+            pass
```

Comment:
Is this still needed? Again IIUC we are not going to support 3.5.1.

Reply:
I would leave it - it does not hurt